thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/MergedAnomalyResultManager.java

```diff
@@ -42,5 +42,7 @@ MergedAnomalyResultDTO findLatestConflictByFunctionIdDimensions(Long functionId,
 
   List<MergedAnomalyResultDTO> findByTime(long startTime, long endTime);
 
+  List<MergedAnomalyResultDTO> findUnNotifiedByFunctionIdAndIdLesserThanAndEndTimeGreaterThanLastOneDay(long functionId, long anomalyId);
```

Comment:
find Not Notified anomalies ByFunctionId (given functionId) And anomaly Id Lesser Than (given anomaly id) And EndTimeGreaterThanLastOneDay